docs/UserGuide.adoc

```diff
@@ -18,30 +18,34 @@ By: `Team T08-3`      Since: `Jan 2019`      Licence: `MIT`
 
 == Introduction
 
-PlanMySem is a CLI (Command Line Interface) scheduling/calendar application that targets NUS students and staff who prefer to use a desktop application for managing their schedule/calendar.
-PlanMySem automatically creates a planner that is synchronised according to the NUS academic calendar for the current semester and enables easy creation, editing and deleting of items.
+Thank you for using _PlanMySem_!
+
+_PlanMySem_ is a CLI (Command Line Interface) scheduling/calendar application that targets NUS students and staff who prefer to use a desktop application for managing their schedule/calendar.
```

Comment:
Perhaps we could change the first line to "PlanMySem is a text-based (Command Line Interface) scheduling/calendar application ...."

Reply:
That's better, thanks.